helm/values.yaml

```diff
@@ -0,0 +1,217 @@
+# Default values for wagtail.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+## Define serviceAccount names for components. Defaults to component's fully qualified name.
+##
+serviceAccounts:
+  wagtail:
+    create: true
+    name: wagtail
+    annotations: {}
+  memcached:
+    create: true
+    name: memcached
+    annotations: {}
+
+# -------------------------------------------------------------
+# WAGTAIL
+# -------------------------------------------------------------
+
+wagtail:
+  name: wagtail
+  image:
+    repository: "ghcr.io/ietf-tools/wagtail_website"
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: "latest"
```

Comment:
Don't override this to the latest.
When publishing a chart we can set the appVersion to be the new image version.
By doing that different helm chart versions can install the corresponding wagtail_website version.